Docs/source/install/hpc/s3df.rst

```diff
@@ -0,0 +1,189 @@
+* Filesystem locations:
+    * User folder: ``$HOME>`` (25GB on Weka file system)
+    * Scratch folder: ``/sdf/scratch/<username_initial>/<username>`` (100GB on Weka file system)
+    * Group storage: Depends on your group, see e.g. information for the ATLAS group `here <https://usatlas.readthedocs.io/projects/af-docs/en/latest/sshlogin/ssh2SLAC/>`__
+
+Through S3DF we have access to CPU and GPU nodes (the latter equipped with NVIDIA A100).
+
+
+
+Installation
+------------
+
+
+1) Prerequisites
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+
+- S3DF login to a node with access to A100 GPUs (via SLURM job or interactive session).
+- Conda available in your shell (Miniconda/Anaconda on S3DF user space).
+- GPU: NVIDIA A100 (sm_80).
+
+For other GPUs, adjust ``-DCMAKE_CUDA_ARCHITECTURES``:
+
+- V100: `70`
+- A100: `80`
+- H100: `90`
+
+.. note::
+
+  SLURM tip (S3DF):
+
+  -  Use srun to have GPU nodes allocated. WarpX requires a "task" per CPU-GPU pair, so if you want to run WarpX e.g. on 4 GPUs, you would do:
+
+  .. code-block:: bash
+
+    srun --pty --cpus-per-task=8 --ntasks-per-node=4 --gpus=4 --mem=100GB --nodes=1 --time=2:00:00 --partition=ampere --account=atlas bash
+
+
+
+
+2) Create & Populate the Conda Environment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+
+  .. code-block:: bash
+
+    # A) create env (example set; versions can vary by site repos)
+
+    conda create -n warpx-gpu-test -y python=3.11
+    conda activate warpx-gpu-test
+
+    #Get the full CUDA toolkit in your conda env.
+    conda install -c nvidia -y "cuda=12.9.*"
+
+    # Additionally install NVTX dev tools (cuda separates from the runtime version)
+    conda install -c nvidia -y "cuda-nvtx-dev=12.9.*"
+
+    # B) Install build tools
+    conda install -c conda-forge -y \
+      "cmake>=3.27" ninja make pkg-config git \
+      openmpi ucx mpi4py \
+      gcc_linux-64=13 gxx_linux-64=13
+
+    # C) Install ADIOS2 with OpenMPI (so we can use openpmd_backend = bp)
+    conda install -c conda-forge "adios2=*=mpi_openmpi*"
+
+    # D) Verify build tools
+    which cmake && cmake --version     # should show cmake >= 3.24 from your env
+    which nvcc  && nvcc  --version     # should show CUDA 12.9 from the nvidia 'cuda' metapackage
+    which mpicxx && ompi_info | head   # Open MPI from conda-forge
+
+
+
+.. note::
+  - ``cuda-nvtx-dev`` provides the nvtx3 headers used by recent NVTX versions.
+  - ``cuda-nvcc`` brings nvcc under ``$CONDA_PREFIX/bin/nvcc``.
+  - MPI here is via ``mpich`` in the env (``mpicc``, ``mpicxx`` will be at ``$CONDA_PREFIX/bin/...``).
+
+
+3) Download WarpX
+^^^^^^^^^^^^^^^^^
+
+.. code-block:: bash
+
+  git clone https://github.com/BLAST-WarpX/warpx.git
+  cd warpx
+
+
+
+
+4) Environment Fixes (NVTX headers + dlopen/dlsym)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+
+Recent NVTX installs the header under ``nvtx3/nvToolsExt.h``. Add it to your include path, and ensure ``libdl`` is linked (``CUDA/NVTX`` uses ``dlopen()/dlsym()`` dynamically):
```

Comment:
Interesting, I do not need these NVTX quirks once the `cuda-nvtx-dev` package is installed:
https://github.com/BLAST-ImpactX/impactx-benchmarks/blob/main/napac25/benchmark-gpu-conda.yaml